.circleci/config.yml

```diff
@@ -657,6 +658,7 @@ jobs:
                 --config $PWD/nipype.cfg -w /tmp/ds210/work \
                 /tmp/data/ds210 /tmp/ds210/derivatives participant \
                 --fs-no-reconall --sloppy --write-graph \
+                --output-spaces MNI152NLin2009cAsym \
```

Comment:
Why on for anatomical, off for full run? And isn't this one of the default output spaces?